### PR TITLE
Add external subtitle tracks to sample apps

### PR DESCRIPTION
**Pull request by lukasf**
This should solve issue #75.

### CODE REVIEW

**Comment by lukasf** [comment]
I also cleaned up a few things in the samples (remove unneccessary null checks).

File: Samples/MediaPlayerCS/MainPage.xaml.cs [hunk 1]
```diff
@@ -195,6 +168,46 @@ private async void ExtractFrame(object sender, RoutedEventArgs e)
...
         }
 
+        private async void LoadSubtitleFile(object sender, RoutedEventArgs e)
+        {
+            if (playbackItem != null)
+            {
+                FileOpenPicker filePicker = new FileOpenPicker();
+                filePicker.ViewMode = PickerViewMode.Thumbnail;
+                filePicker.SuggestedStartLocation = PickerLocationId.VideosLibrary;
+                filePicker.FileTypeFilter.Add("*");
```
Comment (touseefbsb) [inline Samples/MediaPlayerCS/MainPage.xaml.cs]: maybe the filter type should be limited to all those subtitle file types which are supported by uwp apis ?

Comment (mcosmin222) [reply Samples/MediaPlayerCS/MainPage.xaml.cs]: Do you have such a list? There is none available from docs afaik.

Comment (lukasf) [reply Samples/MediaPlayerCS/MainPage.xaml.cs]: This is just a sample, it shows how to do things in a very basic way. For a real app, this needs to be done differently anyways. Like, you'd have to provide the possibility to also select two files, for case of image based subtitles, and use different APIs to create the TimedTextSource. I do not plan to add all that in the sample.

Comment (mcosmin222) [reply Samples/MediaPlayerCS/MainPage.xaml.cs]: I agree, this doesn't have to be production ready code. You might even want to load the subtitle automatically on media open. Or have custom pickers.

**Pull request merged**